nnvm/python/nnvm/frontend/coreml.py

```diff
@@ -249,6 +259,15 @@ def PermuteLayerParams(op, insym, symtab):
     axes = tuple(op.axis)
     return _sym.transpose(insym, axes=axes)
 
+def UpsampleLayerParams(op, insym, symtab):
+    if op.scalingFactor[0] != op.scalingFactor[1]:
```

Comment:
@srkreddy1238 
For coreml, the input dimensions are not available in frontend. So we wont be able to resize based on shape, and i feel using upsample is better here.
Regarding the shape, since the data is coming from the previous layer/Op of TVM, it will be in the expected format. And if the data format is wrong, during build process, tvm will throw error.